content/sdk/10.js/00.ethers/20.v6/10.types.md

```diff
@@ -1,23 +1,36 @@
 ---
 title: Types and Interfaces
-description:
+description: Definitions and interfaces used in ZKsync Era.
+tags: ["ZKsync", "types", "interfaces", "Ethereum", "blockchain"]
 ---
 
 ## `AccountAbstractionVersion`
 
 Enumerated list of account abstraction versions.
 
-- None = `0` - Used for contracts that are not accounts.
-- Version1 = `1` - Used for contracts that are accounts.
+::collapsible
+  ::field{name="None" type="number" defaultValue="0"}
+  Used for contracts that are not accounts.
+  ::
+  ::field{name="Version1" type="number" defaultValue="1"}
+  Used for contracts that are accounts.
+  ::
+::
```

Comment:
Same with these properties for this page, please double check all of them to ensure they're formatted correctly.